data/tome_of_beasts_3/monsters.json

```diff
@@ -12637,7 +12637,7 @@
   "page_no": 168
  },
  {
-  "name": "Angel, Archangel Iilari\u2019Jil",
+  "name": "Angel, Archangel IilariJil",
```

Comment:
Here and on L23100, it looks like you've accidentally removed the ’ character. (\u2019 is the escape code for right single quotation.) If your intention was to remove the escape character, maybe we could use an apostrophe instead? Iilari'Jil would be more proper.

Reply:
Yep, thank you.